torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -75,7 +75,7 @@ RegisterOperators reg({
           };
         }),
     Operator(
-        prim::TensorToNum,
+        "prim::TensorToNum(Tensor a) -> Scalar",
```

Comment:
I'm not that familiar with how the schemas will be used, but it might be better to split this up in to TensorToInt and TensorToFloat. As it currently stands, the output value of the node will have the accurate Scalar subtype but the schema doesn't.

Reply:
Okay sure.  Now that we have schema's for these nodes should we be using graph.insert instead of graph.create ... ?